packages/nimble-components/src/table/tests/table.pageobject.ts

```diff
@@ -25,7 +26,25 @@ export class TablePageObject<T extends TableRecord> {
             );
         }
 
-        return headers.item(columnIndex).textContent?.trim() ?? '';
+        return this.getHeaderContentElement(headers[columnIndex]!);
+    }
+
+    public getRenderedHeaderContent(
```

Comment:
nit: the function names aren't really helping me understand the difference between `getRenderedHeaderContent` and `getHeaderContent` at a glance. Without fully understanding the DOM structure I don't have great suggestions for improvement. Maybe `getHeaderContent` and `getHeaderContentFirstChild`? If you don't like that and nobody has better ideas I can live with the current names.

Reply:
Removed.